pkg/viewmodels/content_object.go

```diff
@@ -54,9 +54,8 @@ func GetTitleInformationByContentObject(ctx context.Context, fetch *dsmodels.Fet
 	// AgendaItemNumber
 	switch result.Collection {
 	case "assignment":
-	case "motion":
 	case "motion_block":
-	case "topic":
+	case "topic", "motion":
```

Comment:
```suggestion
	case "assignment", "topic", "motion", "motion_block":
```

While at it also update the other cases.